engine/runtime-integration-tests/src/test/scala/org/enso/compiler/test/pass/desugar/FunctionBindingTest.scala

```diff
@@ -287,7 +287,7 @@ class FunctionBindingTest extends CompilerTest {
     "work recursively" in {
       val ir =
         """
-          |f (a = (f a = a)) =
```

Comment:
This case doesn't make any sense and just seems to be a previously-undetected syntax error.